SPECS/python-parallel-ssh/python-parallel-ssh.spec

```diff
@@ -0,0 +1,131 @@
+%global srcname parallel-ssh
+
+Name:           python-%{srcname}
+Version:        1.9.1
```

Comment:
This is a fairly old version. Can we upgrade to a newer version?

Reply:
Bumped version to 2.10.0